composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "hybridauth/hybridauth",
+    "name": "mourjan/hybridauth",

```

Comment:
Done..

am I supposed to submit another Pull Request?

Bassel

On Fri, Jul 8, 2016 at 4:37 AM, Jan Dvorak <notifications@github.com> wrote:

> In composer.json
> <https://github.com/hybridauth/hybridauth/pull/658#discussion_r70015151>:
>
> > @@ -1,5 +1,5 @@
> >  {
> > -    "name": "hybridauth/hybridauth",
> > +    "name": "mourjan/hybridauth",
>
> Please fix this back.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hybridauth/hybridauth/pull/658/files/e9a99e685ec1b7bc8f3a93ef82172cadaf7d0f57#r70015151>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFzgxk6J2tf0xy3MVCi-RitIknLKP2VHks5qTanpgaJpZM4JDFP3>
> .
>